scripts/setup.py

```diff
@@ -267,47 +273,79 @@ def rename_project(self):
                     self.replace_text(path, old_project_name, self.project.new_project_name)
             print(f'✅  Renamed to {self.project.new_project_name} in Flutter succesfully!')
         elif old_project_name is None:
-            print("Unable to update project name in Flutter! Please check again!")
+            print("❌ Unable to update project name in Flutter! Please check again!")
             sys.exit()
         else:
             print("Reusing old project name in Flutter!")
 
+    def update_project_version(self):
+        current_project_version = self.get_current_project_version()
+        new_project_version = f"{self.project.app_version}+{self.project.build_number}"
+        if current_project_version is not None and current_project_version != new_project_version:
+            pubspec_file = self.project.project_path + os.sep + "pubspec.yaml"
+            self.replace_text(pubspec_file, current_project_version, new_project_version)
+            print(
+                f'✅  Updated project version to {new_project_version} in Flutter succesfully!')
+        elif current_project_version is None:
+            print("❌ Unable to update project version in Flutter! Please check again!")
+            sys.exit()
+        else:
+            print("Reusing old project version in Flutter!")
+
     def run(self):
         self.rename_project()
+        self.update_project_version()
 
 
 def handleParameters():
     parser = argparse.ArgumentParser(
-        description='The necessary arguments for renaming package and project.'
+        description='The necessary arguments for initializing the project.'
     )
     parser.add_argument('--project_path',
                         type=str,
                         required=True,
                         help='The project path')
     parser.add_argument('--package_name',
                         type=str,
-                        default='co.nimblehq.flutter.template',
+                        default='',
                         help='The new package name')
     parser.add_argument('--app_name',
                         type=str,
-                        default='Flutter Templates',
+                        default='',
                         help='The app name')
     parser.add_argument('--project_name',
                         type=str,
-                        default='flutter_templates',
+                        default='',
                         help='The project name')
```

Comment:
Should we put `required=True,` for package name, project name, and app name parameters? 🤔

Reply:
@luongvo make sense, will update 🙏